platform/android/src/style/sources/source.cpp

```diff
@@ -93,6 +93,15 @@ namespace android {
         return attribution ? jni::Make<jni::String>(env, attribution.value()) : jni::Make<jni::String>(env,"");
     }
 
+    void Source::setPrefetchZoomDelta(jni::JNIEnv&, jint delta) {
+        source.setPrefetchZoomDelta(delta);
+    }
+
+    jint Source::getPrefetchZoomDelta(jni::JNIEnv&) {
+        auto delta = source.getPrefetchZoomDelta();
+        return delta ? delta.value() : 0;
+    }
+
```

Comment:
setter and getter have 'optional' semantics, could this be reflected in Android API?

With current change, once the client sets delta, there is no way to roll-back to default behavior, e.g., calling `Source::setPrefetchZoomDelta(nullopt)`, same applicable for getter.